grpc/execution/test_setup.go

```diff
@@ -0,0 +1,12 @@
+package execution
+
+import (
+	"github.com/ethereum/go-ethereum/grpc/shared"
+	"testing"
+)
+
+func SetupExecutionService(t *testing.T, sharedService *shared.SharedServiceContainer) *ExecutionServiceServerV1 {
```

Comment:
don't think this needs to be exported either

Reply:
hmm it is being used in the optimistic service tests to call `GetGenesisInfo` and `GetCommitmentInfo`, so i do need to export it. Maybe I can restructure the tests better